open-media-match/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -106,12 +107,7 @@ def lookup():
     return enabled_banks
 
 
-@bp.route("/index/status")
-def index_status():
-    """
-    Input:
-     * Signal type (hash type)
-    Output:
-     * Time of last index build
-    """
-    abort(501)  # Unimplemented
+@bp.route("/index/<index_type_name>/status")
+def index_status(index_type_name):
```

Comment:
add a type of str to index_type_name for mypy

```
index_status(index_type_name: str)
```

Reply:
Done